tools/dtrace/README.md

```diff
@@ -263,24 +263,53 @@ Trace a downstairs IO and measure time for in in the following three parts:
 * 3rd report is OS done to downstairs sending the ACK back to upstairs
 
 ## upstairs_info.d
-This is a dtrace script for printing some simple upstairs state info.
+This is a dtrace script for printing upstairs state and work queue info.
 If the upstairs is not yet running, add the -Z flag to dtrace so it will
 wait to find the matching probe.
 ```
 pfexec dtrace -s upstairs_info.d
 ```
 
 You start crucible, then run the above script.  Output should start appearing
-right away with the state of the three downstairs and a count of active
-jobs for upstairs and downstairs.
+with a few seconds.
```

Comment:
extreme nit moment: probably "within"

Reply:
Neither extreme nor a nit, that's a straight up error!